functional-tests/app/app.js

```diff
@@ -306,3 +307,58 @@ search.addWidget(
 );
 
 search.start();
+
+const logs = {};
+/**
+ * This function is a JS client factory that can be used to record
+ * queries used in the player.
+ *
+ * In order to record new queries:
+ *  - replace player by recorder
+ *  - run the app, and do some queries
+ *  - use your console and do `copy(logs)`
+ *  - paste the content in the coldQueries
+ *
+ * @param {function} algoliasearch original algoliasearch factory
+ * @param {string} app appID
+ * @param {key} key API Key
+ * @return {object} a JS client
+ */
+function recorder(algoliasearch, app, key) { // eslint-disable-line
+  const client = algoliasearch(app, key);
+  const searchƒ = client.search;
```

Comment:
`ƒ` is probably from copy pasting?

Reply:
No it's on purpose to differentiate from the other `search`

Reply:
I am used when monkey patching to write:

```js
var originalSearch = client.search
```

WDYT?

Reply:
Ok. If 2 devs ask questions better change ;)